tests/defined_resource_types_spec.rb

```diff
@@ -38,13 +62,23 @@
   end
 end
 describe "Task 3:", host: :localhost do
-  it 'Create an ssh key' do
+  it 'has a working solution', :solution do
+    command("echo | ssh-keygen -t rsa -P puppet")
```

Comment:
@greglarkin Yeah, I noticed that too. It's good to be aware of, but I don't think it should be a problem.